src/components/accession2/Accession2View.tsx

```diff
@@ -46,6 +60,46 @@ export default function Accession2View(): JSX.Element {
         </Typography>
         <Typography color='#708284'>{accession?.speciesCommonName}</Typography>
       </Box>
+
+      <Box display='flex'>
+        {accession?.state && (
+          <Box display='flex' padding={(theme) => theme.spacing(0, 3)}>
+            <Icon name='seedbankNav' className={classes.iconStyle} />
+            <Typography paddingLeft={1}>{accession.state}</Typography>
+          </Box>
+        )}
+        {accession?.storageLocation && (
+          <Box display='flex' padding={(theme) => theme.spacing(0, 2)}>
+            <Icon name='info' className={classes.iconStyle} />
+            <Typography paddingLeft={1}>{accession.storageLocation}</Typography>
+          </Box>
+        )}
+        <Box display='flex' padding={(theme) => theme.spacing(0, 2)}>
+          <Icon name='notification' className={classes.iconStyle} />
+          <Typography paddingLeft={1}>Notification</Typography>
```

Comment:
strings

Reply:
The text here is an actual notification/reminder "End-drying Reminder in 25 days" for example. This is just to see how it looks

Reply:
Ok, got it.